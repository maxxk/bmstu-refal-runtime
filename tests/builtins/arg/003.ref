$ENTRY Go
{
	= <Prout <Arg 1>>
}
