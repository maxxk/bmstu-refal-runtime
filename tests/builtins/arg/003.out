[Builtins][Arg]: Bad program argument number! Program args count: 1. Passed arg num: 1. But must in range [1, 1).
